sdk/agrifood/azure-agrifood-farming/samples/README.md

```diff
@@ -0,0 +1,86 @@
+---
+page_type: sample
+languages:
+  - python
+products:
+  - azure
+  - azure-farmbeats
+urlFragment: farmbeats-samples
+---
+
+# Azure FarmBeats samples for Python client
+
+These sample programs demonstrate some common use case scenairos for the [FarmBeats Python client library][python_sdk].
+
+- [sample_hello_world.py][hello_world_sample] demonstrates the most basic operation that can be performed - creation of a Farmer. Use this to understand how to create the client object, how to authenticate it, and make sure your client is set up correctly to call into your FarmBeats endpoint.
+
+- [sample_attachments.py][attachments_sample] demonstrates FarmBeats' capabaility of storing arbitrary files in context to the various [farm hierarchy][farm_hierarchy_docs] objects. We first attach some files onto a farmer and a farm, and then download all existing attachments for the farmer onto a local directory.
+
+Additionally, for each sample, there are corresponding files in the [`samples/async`][async_samples] directory
+
+## Prerequisites
+
+To run the samples, you need:
+
+- A [python][get_python] environment. Supported versions are 2.7 and 3.6+.
+- An Azure subscription. Create a free subscription [here][azure_free_sub].
+- A FarmBeats resource. See [installation docs][install_farmbeats] to create a new FarmBeats resource.
+
+Additionally, there are some specific prerequisites if you want to leverage third party integrations into FarmBeats:
+
+- A subscription to a [supported weather provider][weather_docs], to run the weather sample.
+
+## How to run the samples
+
+### Install the dependencies
+
+To run the samples, you need the following three dependencies.
+```bash
+pip install azure-agrifood-farming azure-identity asyncio
+```
+
+### Set up the credentials for authentication
```

Comment:
Isn't it better to let people know at least one of the happy paths rather than them hitting into errors, and then correcting them? At least that's the thought from our side.

Reply:
Confirmed with my team, we want to keep this, added a short blurb pointing to the docs.